service/complete_swap_transaction.js

```diff
@@ -65,13 +65,13 @@ module.exports = (args, cbk) => {
     },
 
     // Initialize the LN daemon connection
-    lnd: cbk => {
+    lnd: ['validate', ({}, cbk) =>  {
       try {
-        return cbk(null, lightningDaemon({}));
+        return cbk(null, lightningDaemon({network}));
```

Comment:
Looks like this would fail a test because it would need to be `{network: args.network}`